dts/arm/st/u5/stm32u599.dtsi

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023 PSICONTROl nv
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/u5/stm32u5.dtsi>
+
+
+/ {
+	soc {
+		pinctrl: pin-controller@42020000 {
```

Comment:
If possible define the stm32u595.dtsi (instead of stm32u599 here) and  then include the stm32u599 from the stm32u595.
```
Device subfamily
599 = STM32U599xx with GFX accelerators and interfaces, OTG_HS
595 = STM32U595xx with OTG_HS
```

Reply:
Done